chromium_src/third_party/blink/renderer/core/exported/web_view_impl.cc

```diff
@@ -15,10 +15,8 @@
   {                                                        \
     RuntimeEnabledFeatures::SetBraveIsInTorContextEnabled( \
         prefs.is_tor_window);                              \
-    if constexpr (BUILDFLAG(IS_MAC)) {                     \
-      if (prefs.is_tor_window) {                           \
-        RuntimeEnabledFeatures::SetWebShareEnabled(false); \
-      }                                                    \
+    if (prefs.is_tor_window) {                             \
```

Comment:
@fmarier We can check Window version here and disable the web share only on 11 or newer, is it important?

Reply:
No, let's just disable it entirely for all Tor windows. It's not really worth keeping this feature in that mode.